src/test/scala/chiselTests/ModulePrefixSpec.scala

```diff
@@ -357,4 +375,27 @@ class ModulePrefixSpec extends ChiselFlatSpec with ChiselRunners with Utils with
 
     matchesAndOmits(chirrtl)(lines: _*)()
   }
+
+  it should "omit the prefix if localModulePrefixUseSeparator is false" in {
+    class Foo extends RawModule {
+      override def localModulePrefix = Some("Inner")
+      override def localModulePrefixUseSeparator = false
+    }
+    class Top extends RawModule {
```

Comment:
should we add a test if a Parent with localModulePrefixUseSeparator = false instantiates a child with it at the default `true` and confirm you get something like `ParentChild_Grandkid`?

Reply:
Good idea will add

Reply:
Done.